src/Potree.js

```diff
@@ -566,17 +566,13 @@ Potree.updateVisibility = function(pointclouds, camera, renderer){
 			// TODO
 			window.warned125 = true;
 		}
-		if(false && pointcloud.material.clipBoxes.length > 0){
```

Comment:
Nice conditional 😆

Reply:
There are many examples of nice code like this 😰